### PR TITLE
Update manage.md

### PR DESCRIPTION
**Pull request by msbemba**
Added delegated admin role to the document as one needs to have the admin role to be able to switch tenants.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8157

File: microsoft-365/admin/multi-tenant/manage.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ description: "Learn how to use the tenant switcher and multi-tenant views that g
 
 # Multi-tenant management
 
-Multi-tenant management offers a unified form of management that allows Microsoft 365 partner admins the ability to administer all the tenants they manage from a single location. If you're a partner who manages multiple tenants, you can:
+Multi-tenant management offers a unified form of management that allows Microsoft 365 partner admins the ability to administer all the tenants they manage from a single location. If you're a partner who has delegated admin role and manages multiple tenants, you can:
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/multi-tenant/manage.md]:
```suggestion
Multi-tenant management offers a unified form of management that allows Microsoft 365 partner admins the ability to administer all the tenants they manage from a single location. If you're a partner who has a delegated admin role and manages multiple tenants, you can:
```

**Review by JohanFreelancer9 — APPROVED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@msbemba, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**